pages/load-balancer/reference-content/http2-http3.mdx

```diff
@@ -40,9 +40,9 @@ Backends and frontends of Scaleway Load Balancers support HTTP/1, HTTP/2 and HTT
 
 |        | Frontend with TLS | Frontend without TLS | Backend with TLS | Backend without TLS |
 |--------|:-------------------:|:----------------------:|:------------------:|:---------------------:|
-| HTTP/1 | <Icon name="validate"/>                 | <Icon name="validate"/>                    | <Icon name="validate"/>                | <Icon name="validate"/>                   |
-| HTTP/2 | <Icon name="validate"/>                 | ❌                    | <Icon name="validate"/>                | ❌                   |
-| HTTP/3 | <Icon name="validate"/>                 | ❌                    | ❌                | ❌                   |
+| HTTP/1 | Yes <Icon name="validate"/>                 | Yes <Icon name="validate"/>                    | Yes <Icon name="validate"/>                | Yes <Icon name="validate"/>                   |
+| HTTP/2 | Yes <Icon name="validate"/>                 | No ❌                    | Yes <Icon name="validate"/>                | No ❌                   |
```

Comment:
I think we said to remove any emojis - maybe just keep the text version

Reply:
I spoke to Jess about it and she agreed it can be OK to leave the emojis as long as the text is there too
I think it's easier to read at a glance with the emojis, especially for users who might struggle taking in large blocks of test